app/templates/agenda/agendaitems/agendaitem/index.hbs

```diff
@@ -1,3 +1,23 @@
+{{#let this.model.treatment.decisionActivity.decisionResultCode as |drc|}}
+  {{#if drc.isRetracted}}
+    <Auk::Alert @skin="warning" @message={{t "retracted-item"}}/>
+  {{/if}}
+  {{#if drc.isPostponed}}
```

Comment:
`{{else if}}`?

Reply:
since there is only 1 status possible I didn't think there was a need for an `else if` here.